py/worker/binary/fvs_worker.py

```diff
@@ -52,28 +46,14 @@ def driver_fn(sight):
   decision.decision_outcome('outcome_label', sight, reward=0, outcome=outcome)
   # return sight
 
-
-#temporary created
-def get_sight_instance():
-  params = sight_pb2.Params(
-      label=get_question_label(),
-      bucket_name=f'{os.environ["PROJECT_ID"]}-sight',
-  )
-  sight_obj = Sight(params)
-  return sight_obj
-
-
 def main(argv: Sequence[str]) -> None:
   if len(argv) > 1:
     raise app.UsageError("Too many command-line arguments.")
 
-  with get_sight_instance() as sight:
-    decision.run(
-        driver_fn=driver_fn,
-        sight=sight,
-        #  action_attrs=fvs_api.get_action_attrs(),
-        #  outcome_attrs=fvs_api.get_outcome_attrs(),
-        question_label=get_question_label())
+  with Sight.create(get_question_label()) as sight_obj:
```

Comment:
since Sight is created using the constructor that only uses the question label can we change worker_main_function to create the Sight object on its own if it is not provided (make it a default=None) parameter.

In this case the worker code becomes:
if __name__ == "__main__":
  sight.worker_main_function(label, driver)